pages/api/interaction_work.ts

```diff
@@ -47,6 +47,10 @@ export interface BlockActionInteraction {
     text: string;
     ts: string;
     thread_ts?: string;
+    edited?: {
+      user: string;
+      ts: string;
+    }
```

Comment:
Can you clarify where this is used? It isn't immediately apparent in this PR

Reply:
It's used on line 95 of block_action.ts